app/controllers/nominate_practices_controller.rb

```diff
@@ -1,15 +1,12 @@
 class NominatePracticesController < ApplicationController
+
   def index
-    # TODO remove once we add captcha to form
-    redirect_to root_path if helpers.is_user_a_guest?
   end
 
   def email
-    # TODO remove once we add captcha to form
-    if helpers.is_user_a_guest?
-      respond_to do |format|
-        format.html { redirect_to root_path }
-      end
+    if params["phone"].to_s.length > 0
```

Comment:
I think we should change the param name to "phony" if that's what we're using for the column in the DB. It may be a little confusing down the line.

Reply:
I can change the db name to phone... we want to keep the name 'phone' to look like a legit field.